src/FluidProposals.sol

```diff
@@ -284,8 +293,16 @@ contract FluidProposals is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     }
 
     function _removeProposal(uint256 _proposalIndex) internal {
+        // console.log("Removing proposalIndex %d", _proposalIndex);
         uint256 proposalId = activeProposals[_proposalIndex];
-        superfluid.deleteFlow(token, registeredProposals[proposalId].beneficiary);
+        // console.log("Proposal id %d", proposalId);
+        try superfluid.deleteFlow(token, registeredProposals[proposalId].beneficiary){
+            // console.log("Flow deleted");
+        }catch(bytes memory error){
```

Comment:
Flow not created error can happens by Superfluid side that can revert here without try catch and DoS the contract